src/genlab_bestilling/models.py

```diff
@@ -506,7 +506,7 @@ class Meta:
     objects = managers.SampleAnalysisMarkerQuerySet.as_manager()
 
     def __str__(self):
-        return f"{str(self.sample)} {str(self.marker) @ {str(self.order)}}"
+        return f"{str(self.sample)} {str(self.marker) @ {str(self.order)}}"  # FIXME What is this syntax?
```

Comment:
You mean that it should not be necessary to call str(...) within an f-string? It is possible to use `!s` as an alternative. I am not sure which method is called by default, if `str`, `repr` or something else.

Reply:
Oh, right, I missed it. I guess it is a typo.

Reply:
Yes please.

Reply:
Done